src/extensions/MarketCreation.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.4.13;
+
+
+// FIXME: Remove this stub once the contract is implemented in Solidity
+contract Market {
+    function stub();
+}
+
+
+// FIXME: Remove this stub once the contract is implemented in Solidity
+contract ReportingWindow {
+    function createNewMarket(int256 _endTime, int256 _numOutcomes, int256 _payoutDenominator, int256 _feePerEthInWei, address _denominationToken, address _sender, int256 _minDisplayPrice, int256 _maxDisplayPrice, address _automatedReporterAddress, int256 _topic) payable returns (Market);
+}
+
+
+// FIXME: Remove this stub once the contract is implemented in Solidity
+contract Branch {
+    function getReportingWindowByMarketEndTime(int256 _endTime, int256 _hasAutomatedReporter) returns (ReportingWindow);
+}
```

Comment:
Recommend importing `reporting/Interfaces.sol` for these (you'll need to rebase/merge develop).